src/Routes.js

```diff
@@ -47,8 +48,12 @@ class App extends Component {
                   render={() => <Redirect to="/" />}
                 />
                 <Route
-                  path="/:topicSlug/:proposalSlug"
-                  render={routeProps => <Proposal signaling {...routeProps} />}
+                  path="/polling-proposal/:topicSlug/:proposalSlug"
```

Comment:
So for this, I've updated the routes for both executive and polling proposals to use `/:execSlug` or `/:pollSlug` respectively. With that, I've removed the topicKey lookup in the reduxProps in both proposal files and will only determine what proposal to render on the basis of the proposal title matching the `:pollSlug` or `:execSlug` in each case.
Should we get to the point that we are using ipfs to host the markdown content, I think it would be a good idea to use the hash as an identifier here.